pallets/identity-management/src/lib.rs

```diff
@@ -96,22 +94,18 @@ pub mod pallet {
 		//       Can we retrieve that extrinsic hash in F/E?
 		IdentityLinked {
 			account: T::AccountId,
-			id_graph_hash: H256,
 			req_ext_hash: H256,
```

Comment:
Would calling the public getter be cheaper than including the hash in the event's response? I can imagine clients calling the public id_graph_hash getter right after each IdentityManagement operation.

Reply:
In short - I don't think client needs to listen to the event, the synchronous DI response contains everything that a client needs.

However, if you prefer having it in parachain event for tracking purpose, I'm fine too

Reply:
>  I can imagine clients calling the public id_graph_hash getter right after each IdentityManagement operation.

The client doesn't have to. It's included in each DI request that mutates the IDGraph, see `trusted_call_result.rs`

Reply:
After a second thought I reckon it's beneficial to include these hashes for tracking purposes.

Reply:
Thank, Kai!